supernode/daemon/mgr/cdn/manager.go

```diff
@@ -51,10 +72,12 @@ type Manager struct {
 	originClient    httpclient.OriginHTTPClient
 	pieceMD5Manager *pieceMD5Mgr
 	writer          *superWriter
+	metrics         *metrics
 }
 
 // NewManager returns a new Manager.
-func NewManager(cfg *config.Config, cacheStore *store.Store, progressManager mgr.ProgressMgr, originClient httpclient.OriginHTTPClient) (*Manager, error) {
+func NewManager(cfg *config.Config, cacheStore *store.Store, progressManager mgr.ProgressMgr,
```

Comment:
Please update the mock cdn manager by exec the script `hack/generate-super-mock-mgr.sh`.

Reply:
This script didn't work. Cannot generate anything. Is it a bug or what should I change?

Reply:
Oops, sorry. You should  exec the script hack/generate-super-mock-mgr.sh only when you update the interface of `mgr*`.